src/Cake.Common/Tools/DotNetCore/Test/DotNetCoreTestSettings.cs

```diff
@@ -22,6 +22,11 @@ public sealed class DotNetCoreTestSettings : DotNetCoreSettings
         public DirectoryPath OutputDirectory { get; set; }
 
         /// <summary>
+        /// Gets or sets the XML output file
+        /// </summary>
+        public FilePath XmlPath { get; set; }
+
```

Comment:
No, this is a standard `dotnet test` attribute. I was attempting to set up a build script that takes the XML output results, but the only way to get output results currently is by doing `dotnet test` manually with the required parameters which doesn't look too pretty...  so I made this pull request that adds the missing parameter.